universal-login-example/src/views/Modals/BackupModal.js

```diff
@@ -0,0 +1,39 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+class BackupModal extends Component {
```

Comment:
I think it should be called `BackupModalView`. While there should be perhaps other component `BackupModal` that uses `BackupModalView`.

Reply:
Reviewed ✅